tests/helm_install.sh

```diff
@@ -2,77 +2,76 @@
+  --set opentelemetry-operator.manager.image.tag=0.115.0-ubi \
   --set otelcolInstrumentation.statefulset.image.repository=public.ecr.aws/sumologic/sumologic-otel-collector@sha256 \
-  --set otelcolInstrumentation.statefulset.image.tag=fd5c8b496f522ae91279ff96ef977d12815c55e2b15519a27705f27286507bcb \
+  --set otelcolInstrumentation.statefulset.image.tag=4c575848287e41884ce580c4668996f7f4bfe06e9fcc332a9bea99abf0af8791 \
   --set otelevents.image.repository=public.ecr.aws/sumologic/sumologic-otel-collector@sha256 \
-  --set otelevents.image.tag=fd5c8b496f522ae91279ff96ef977d12815c55e2b15519a27705f27286507bcb \
+  --set otelevents.image.tag=4c575848287e41884ce580c4668996f7f4bfe06e9fcc332a9bea99abf0af8791 \
   --set otellogs.daemonset.initContainers.changeowner.image.repository=public.ecr.aws/sumologic/busybox@sha256 \
   --set otellogs.daemonset.initContainers.changeowner.image.tag=209bc8e0b1d958fc699f3bb75b1248544e372ad35cdc46c991bd1698f641e1f9 \
   --set otellogs.image.repository=public.ecr.aws/sumologic/sumologic-otel-collector@sha256 \
-  --set otellogs.image.tag=fd5c8b496f522ae91279ff96ef977d12815c55e2b15519a27705f27286507bcb \
+  --set otellogs.image.tag=4c575848287e41884ce580c4668996f7f4bfe06e9fcc332a9bea99abf0af8791 \
   --set pvcCleaner.job.image.repository=public.ecr.aws/sumologic/kubernetes-tools-kubectl@sha256 \
-  --set pvcCleaner.job.image.tag=89ae5ef9c85f6a01d520c8e237e614131cdc81a09f2eade54c9b091b2993e856 \
+  --set pvcCleaner.job.image.tag=d3155b864cc74f327adedf7d013a8f0fcb11c2e73a5655b7cafa848824e80172 \
   --set sumologic.metrics.collector.otelcol.image.repository=public.ecr.aws/sumologic/sumologic-otel-collector@sha256 \
-  --set sumologic.metrics.collector.otelcol.image.tag=fd5c8b496f522ae91279ff96ef977d12815c55e2b15519a27705f27286507bcb \
+  --set sumologic.metrics.collector.otelcol.image.tag=4c575848287e41884ce580c4668996f7f4bfe06e9fcc332a9bea99abf0af8791 \
   --set sumologic.metrics.remoteWriteProxy.image.repository=public.ecr.aws/sumologic/nginx-unprivileged@sha256 \
-  --set sumologic.metrics.remoteWriteProxy.image.tag=9aef85eed423d0bf6bce16eb0dce3d56d18c0aa7929627ed4fe7aef7aa749c1f \
+  --set sumologic.metrics.remoteWriteProxy.image.tag=7260555d16c94c4fe1e016626b4cb8a5c0de6b700e2eea28744331fb298f71ae \
   --set sumologic.otelcolImage.repository=public.ecr.aws/sumologic/sumologic-otel-collector@sha256 \
-  --set sumologic.otelcolImage.tag=fd5c8b496f522ae91279ff96ef977d12815c55e2b15519a27705f27286507bcb \
+  --set sumologic.otelcolImage.tag=4c575848287e41884ce580c4668996f7f4bfe06e9fcc332a9bea99abf0af8791 \
   --set sumologic.setup.job.image.repository=public.ecr.aws/sumologic/kubernetes-setup@sha256 \
-  --set sumologic.setup.job.image.tag=21819dcc791144843ebed17abf7304e2cefd711995027a8737f2d9ae14418811 \
+  --set sumologic.setup.job.image.tag=f07eaf68fa11ffc708e4485a83bd3763fc5712f7d73d29edfe5b9c9e08f05df6 \
```

Comment:
These SHAs are not from openshift they're from AWS ECR. You could just use the public tags instead

Reply:
You'll need to login to the AWS opensource account to see these SHAs